src/nvim/window.c

```diff
@@ -487,7 +487,7 @@ static void cmd_with_count(char *cmd, char_u *bufp, size_t bufsize,
   size_t len = xstrlcpy((char *)bufp, cmd, bufsize);
 
   if (Prenum > 0 && len < bufsize) {
-    vim_snprintf((char *)bufp + len, bufsize - len, "%" PRId64, Prenum);
+    vim_snprintf((char *)bufp + len, bufsize - len, "%ld", Prenum);
```

Comment:
Just the `Prenum` parameter in the `cmd_with_count` function?  That should be fine.  I'll test it out.